assets/css/custom.scss

```
@@ -5521,3 +5178,746 @@ $cs-mint-faint: rgba(42, 217, 194, 0.06);
     padding-top: 3rem;
   }
 }
+
+// =============================================================================
+// IMMERSIVE CASE STUDY — BODY  (.case-study-immersive)
+// -----------------------------------------------------------------------------
+// The body carries BOTH `case-study-modern` and `case-study-immersive`, so the
+// hero + at-a-glance stat strip reuse the `.case-study-modern .cs-*` rules above
+// (top is byte-identical to modern). EVERYTHING here is scoped to `.csi-*` —
+// never bare element selectors — so it can't touch the modern hero/stats.
+//
```

Stylelint (17.12.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)